### DISCUSSION TITLE
How to get out of "find-definition" buffer without killing it?

### DISCUSSION THREAD

**Discussion by CDitzel** [joaotavora/eglot#815] [Q&A] [answered]
whenever I do a lookup via xref-find-definitions and I want to leave that view again, I do C-k to kill the buffer. 
However, if the buffer that was opened was indeed open before, e.g. in another split, it also closes now. 

So what is the most efficient way to leave the definition view again?

**Comment by Eason0210** [comment, marked as answer]
I use `xref-go-back` to leave the definition view. Does it work for you?

**Reply by CDitzel** [reply]
I dont have this function oO

**Reply by Eason0210** [reply]
It is a bult-in function of `xref.el`. 
I am using Emacs 29.0.50 , it works well.

**Reply by dgutov** [reply]
It was renamed recently. But the binding is the same `M-,`.

**Reply by CDitzel** [reply]
its not, M-, invokes xref-pop-stack oO

**Reply by dgutov** [reply]
Same thing.
